jenkins_jobs/modules/publishers.py

```diff
@@ -527,6 +527,12 @@ def trigger_parameterized_builds(registry, xml_parent, data):
         if any of the property files are not found in the workspace.
         Only valid when 'property-file' is specified.
         (default 'False')
+    :arg bool property-multiline: When enabled properties containing
+        newline character(s) are propagated as TextParameterValue which is
+        a specialized StringParameterValue commonly used for handling
+        multi-line strings in Jenkins. When disabled (default)
+        all properties are propagated as StringParameterValue. (default
```

Comment:
Why do you refer to properties here? Shouldn't they be called parameters?

Maybe the option should be named after the tag name i.e. `text-param-value-on-new-line` to avoid introducing new "concepts"? Especially that another option called `property-file` exists with a completely unrelated meaning.

Reply:
It made sense to name it like that after reading the description. I can change it if you would like.

![image](https://user-images.githubusercontent.com/4595094/88640594-65e39a00-d0be-11ea-867b-3927cf6c9ec8.png)

Reply:
OK no need to change the name. Just please upstream this change to get feedback, especially on the API part.